aleemer/Internify
# Feat/in62 remove persis data

#### User Story
https://chaosneutral.atlassian.net/jira/software/projects/IN/boards/1/backlog?selectedIssue=IN-62

#### Changes made
Made changes to TechStack.js (found in src -> components -> molecules) such that when creating a job posting, if a user unclicks an option (e.g. language C++), that option is removed from the current job object.  A related effect of this is to prevent the same option being added multiple times to the job object (i.e. if a user clicks, unclicks, and then reclicks that option). 

Note that, to be able to view the current job object in Redux, code that was previously included in Create.js, JobPostAction.js, jobPostingReducer.js, and jobPostTypes.js had to be reintroduced.  Leaving it to group consultation whether or not this reintroduced code is desired. 

#### Does your new code introduce new warnings on dev console?
No

#### Test Steps
go into Internify's frontend, into the terminal "npm run dev", attempt to create a job posting starting at the Create page, click F12, and enter the Redux window.  Observe the currentJob object as you select/ unselect options for the job. Should see that persistence has been eliminated as desired.
